charts/lumigo-operator/values.yaml

```diff
@@ -61,4 +61,5 @@ metricsService:
   type: ClusterIP
 endpoint:
   otlp:
-    url: https://ga-otlp.lumigo-tracer-edge.golumigo.com
\ No newline at end of file
+    url: https://ga-otlp.lumigo-tracer-edge.golumigo.com
```

Comment:
@harelmo-lumigo `url` will become the tracing endpoint url?
If so, can we change the name to `traces_url`? (and keep `url` for backward compatibility)